### ISSUE TITLE
nixpkgs is now moving to haskell-ng 

### ISSUE THREAD

**Issue by wavewave** [wavewave/hep-nix-overlay#29]
so we need to upgrade `haskellPackages` to `haskell-ng`.

**Comment by wavewave** [comment]
I've successfully use haskell-ng now in my private development. I think `haskell-ng` is ready to take off.

**Comment by cbpark** [comment]
After pulling the commits in the `master` branch of nixpkgs, I'm experiencing the error message like this:

```
error: attribute ‘cabal’ missing, at hep-nix-overlay/default.nix:221:30
```

when I run commands like `nix-env -qaP | grep something`. The line corresponds to 

```nix
      HEPUtil = callPackage ./pkgs/HEPUtil {
                     cabal = haskellPackages.cabal;
                     inherit haskellPackages;
                     inherit LHCOAnalysis-type;
                   } ;
```

It seems that this is related with the transition into `haskell-ng`. In reality, I found that there is no `nixpkgs.haskellngPackages.cabal`, but `nixpkgs.haskellngPackages.cabal-install`.

**Comment by wavewave** [comment]
yes, that should be replaced. Right now, I am experimenting with haskell-ng packages and soon I will fix the syntax. Thanks.
